portfolio/src/main/webapp/math.js

```diff
@@ -28,19 +30,23 @@ function startGame() {
 
 function check() {
     const answer = document.getElementById("answer");
+    //Compare answers as strings
```

Comment:
Same as above. Perhaps explain why it makes sense to compare the answers as Strings? 
Also, strings should be capitalized.

Reply:
Fixed formatting and elaborated on String comparison.